packages/transport-bridge/src/http.ts

```diff
@@ -181,47 +185,48 @@ export class TrezordNode {
                 },
             ]);
 
-            app.post('/listen', [
+            app.post<ApiPost<Descriptor[]>>('/listen', [
                 parseBodyJSON,
                 (req, res) => {
                     res.setHeader('Content-Type', 'text/plain');
                     this.listenSubscriptions.push({
-                        // todo: type in that if parseBodyText was called as previous handler, req.body is string. is that even possible
-                        // @ts-expect-error
```

Comment:
nice, or definitely nicer than having expect-errors. still the ultimate goal I failed to do was to set req.params and req.body type dynamically based on middleware functions registered before the main handler. eg calling parseBodyText would turn req body from unknown to string.

Reply:
i guess its done now